apps/backend/src/libs/plugins/file-upload/file-upload.plugin.ts

```diff
@@ -0,0 +1,27 @@
+import fastifyMultipart from "@fastify/multipart";
+import fastifyPlugin from "fastify-plugin";
+
+import { FastifyHook } from "~/libs/enums/enums.js";
+
+const fileUpload = fastifyPlugin(async (fastify) => {
+	fastify.decorate("uploadedFile", null);
+
+	await fastify.register(fastifyMultipart);
+
+	fastify.addHook(FastifyHook.PRE_HANDLER, async (request) => {
+		if (request.isMultipart()) {
```

Comment:
```suggestion
		if (!request.isMultipart()) {
		  return;
		}
```
To avoid nested `if` statements